From 37641f246ee9df7289b4e3054b3ded3912773722 Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Tue, 25 Jun 2024 17:32:08 +0200
Subject: [PATCH] hexdump: allow enabling with --disable-all-programs

Currently, if --disable-all-programs is used hexdump cannot be built
as --enable-hexdump is not recognized, so lets add support for it.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 configure.ac | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

--- a/configure.ac
+++ b/configure.ac
@@ -2240,7 +2240,11 @@ UL_BUILD_INIT([column], [check])
 UL_REQUIRES_BUILD([column], [libsmartcols])
 AM_CONDITIONAL([BUILD_COLUMN], [test "x$build_column" = xyes])
 
-UL_BUILD_INIT([hexdump], [yes])
+AC_ARG_ENABLE([hexdump],
+  AS_HELP_STRING([--disable-hexdump], [do not build hexdump]),
+  [], [UL_DEFAULT_ENABLE([hexdump], [check])]
+)
+UL_BUILD_INIT([hexdump])
 AM_CONDITIONAL([BUILD_HEXDUMP], [test "x$build_hexdump" = xyes])
 
 UL_BUILD_INIT([rev], [yes])
